### DISCUSSION TITLE
[Bug]: When using GPT4, the conversation model falls back to GPT3.5.

### DISCUSSION THREAD

**Discussion by orangeagain** [acheong08/ChatGPT#1161] [General]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits

### What happened?

When using GPT4, a warning message "The previous model used in this conversation is unavailable. We've switched you to the latest default model" is seen on the webpage. 
The entire conversation falls back from the black icon to the green icon means it's 3.5 version.

### Steps to reproduce the problem

same code as before
```
from revChatGPT.V1 import Chatbot

chatbot = Chatbot(config={
  "email": "xxx",
  "password": "xxx",
  "access_token": "xxx",
  "session_token": "xxx"
})
conversation_id = "8eaeb9aa-2fc1-456b-a1ac-1fd8a93c88b0"
parent_id = "792a493c-ce54-4f11-a095-6c25dc983e3e"

question = "Remember , the prestigious election of stewardship has strict eligibility criteria. "
for data in chatbot.ask(
    question,
    conversation_id,
    parent_id
):
    response = data["message"]
    
print("result:", response)```

### What should have happened?

Continuing with the 4.0 conversation.

### Version where the problem happens

4.0
from revChatGPT.V1 import Chatbot

### What Python version are you running this with?

_No response_

### What is your operating system ?

_No response_

### Command Line Arguments

```Shell
none, developer mode
```

### Console logs

```Shell
no errors
```

**Comment by acheong08** [comment]
Requires plus account

**Comment by suxianbaozi** [comment]
v4 now is limited beta

**Comment by orangeagain** [comment]
> v4 now is limited beta

I have access to 4


> Requires plus account

I have plus account

**I can access GPT4, I mean I can use gpt4 in Chrome, but use this library access same conversation, the conversation will fallback to DefultMode**

**Comment by orangeagain** [comment]
It's fall back to default model like this:
![89939BC7-D5C1-423C-9160-3475DC311784](https://user-images.githubusercontent.com/12761984/225199673-6d3e6055-88de-4d42-a203-1ffb93140eb7.jpeg)

**Comment by acheong08** [comment]
Did you set the model to `gpt-4` in config?

**Comment by orangeagain** [comment]
> Did you set the model to `gpt-4` in config?

How to setup?

**Comment by acheong08** [comment]
Check readme

**Reply by orangeagain** [reply]
> Check readme

Thank you
